core/common/assets/js/api/modules/command-base.js

```diff
@@ -188,6 +188,16 @@ export default class CommandBase extends ArgsObject {
 		return result;
 	}
 
+	/**
```

Comment:
Please add a description to the commit with details about the technical problem.